benchmark/bench01_distance.jl

```diff
@@ -10,12 +10,12 @@ Random.seed!(2020)
 X = rand(3, 100_000)
 centroids = rand(3, 2)
 d = Vector{Float64}(undef, 100_000)
```

Comment:
This one is wrong, it should be `d = fill(-Inf, 100_000)` and benchmarkable call should be changed to something like
```julia
 @benchmarkable ParallelKMeans.chunk_colwise(d1, $X, $centroids, 1, nothing, 1:100_000, 1) setup=(d1 = copy(d))
```